perllib/Open311/GetServiceRequestUpdates.pm

```diff
@@ -124,16 +126,8 @@ sub update_comments {
                     }
                 );
 
-                if ($request->{media_url}) {
-                    my $ua = LWP::UserAgent->new;
-                    my $res = $ua->get($request->{media_url});
-                    if ( $res->is_success && $res->content_type eq 'image/jpeg' ) {
-                        my $photoset = FixMyStreet::App::Model::PhotoSet->new({
-                            data_items => [ $res->decoded_content ],
-                        });
-                        $comment->photo($photoset->data);
-                    }
-                }
+                $comment = $open311->add_media($request->{media_url}, $comment)
```

Comment:
Not sure why it's returning the object here, it's the same object as was being passed in? (And similarly the other call.)